webview/src/plots/components/customPlots/customPlotsSlice.ts

```diff
@@ -58,7 +55,7 @@ export const customPlotsSlice = createSlice({
       if (!action.payload) {
         return customPlotsInitialState
       }
-      const { plots, colors, ...statePayload } = action.payload
```

Comment:
[Q] @julieg18 why were we sending colors? I can't find any references to it.

Reply:
Leftover from #3575! They can be deleted :)